Services/RequestServer/ConnectionFromClient.h

```diff
@@ -77,4 +77,7 @@ class ConnectionFromClient final
     NonnullRefPtr<Resolver> m_resolver;
 };
 
+// FIXME: Find a good home for this
+ByteString build_resolve_list(DNS::LookupResult const&, StringView host, u16 port);
```

Comment:
`build_curl_resolve_list`, but why does it need to...ah, it's used in the other file too. fine.

Reply:
yeah the duplication in setting up the curl_easy structs makes me a bit sad, but I'll fix that later :tm:. Probably worth a CurlHelpers.h or something.